api/providers.go

```diff
@@ -4,6 +4,11 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/dcos/dcos-diagnostics/collectors"
+	"github.com/dcos/dcos-diagnostics/util"
```

Comment:
Not that we should change this now, but I do want to share this advice: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

I'd like to see us stay away from these types of package names in the future. I'd add 'tools' to the list of bad names as well.